.gitignore

```diff
@@ -19,6 +19,6 @@ erl_crash.dump
 config/dev.exs
 /docs/all.json
 .DS_Store
-/.idea/
+/.idea
```

Comment:
Try not to pollute the code-base with your personal development environment.

This also goes for .DS_Store and .iml